mailpoet/assets/js/src/landingpage/footer.tsx

```diff
@@ -0,0 +1,22 @@
+import { MailPoet } from 'mailpoet';
+import { Button } from 'common';
+import { Heading } from 'common/typography/heading/heading';
+
+function Footer(): JSX.Element {
```

Comment:
I'd refrain from explicitly defining the return type of components and let TS infer the types.

Reply:
Okay.
May I ask why not?

Reply:
Simply put less is better! In a perfect world, TS should be able to infer all types so that we don't need to type hint everything but still enjoy the benefits of a typed language, and it's best to let if infer when it can, unless for some reason we enforce explicitly typing everything via ESLint or alike. Still removing this is only a suggestion.